### PR TITLE
Add Coqtail to CI

### PR DESCRIPTION
**Pull request by whonore**
**Kind:** infrastructure

I didn't see any other IDEs in the CI so I'm not sure if I did all of this correctly or if this is even something desirable. I thought it could be useful to give advance warning if there's a breaking change to the XML protocol.

### CODE REVIEW

File: dev/ci/ci-coqtail.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+ci_dir="$(dirname "$0")"
+. "${ci_dir}/ci-common.sh"
+
+FORCE_GIT=1
+git_download coqtail
+python3 -m pip install pytest
```
Comment (SkySkimmer) [inline dev/ci/ci-coqtail.sh:8]: I think we may prefer to do this in the docker image.

File: dev/ci/ci-coqtail.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+ci_dir="$(dirname "$0")"
+. "${ci_dir}/ci-common.sh"
+
+FORCE_GIT=1
```
Comment (SkySkimmer) [inline dev/ci/ci-coqtail.sh:6]: Why force git?

Comment (whonore) [reply dev/ci/ci-coqtail.sh]: It was just part of the file I initially copied.

File: dev/ci/ci-coqtail.sh (hunk 1 above)
Comment (whonore) [reply dev/ci/ci-coqtail.sh]: I tried adding a line to dev/ci/docker/bionic_coq/Dockerfile but it didn't seem to work. Is there somewhere else I should do it?

Comment (SkySkimmer) [reply dev/ci/ci-coqtail.sh]: It needs admin action, now running.

**Comment by SkySkimmer** [comment]
Looks like it works.

**Review by SkySkimmer — APPROVED** [review]

**Pull request merged**